doc/develop/test/bsim.rst

```diff
@@ -0,0 +1,72 @@
+.. _bsim:
+
+BabbleSim
+#########
+
+BabbleSim and Zephyr
+********************
+
+In the Zephyr project we use the `Babblesim`_ simulator to test some of the Zephyr radio protocols,
+including the BLE stack, 802.15.4, and some of the networking stack.
+
+BabbleSim_ is a physical layer simulator, which in combination with the Zephyr
+:ref:`bsim boards<bsim boards>`
+can be used to simulate a network of BLE and 15.4 devices.
+When we build Zephyr targeting an :ref:`nrf52_bsim<nrf52_bsim>` board we produce a Linux
+executable, which includes the application, Zephyr OS, and models of the HW.
+
+When there is radio activity, this Linux executable will connect to the BabbleSim Phy simulation
+to simulate the radio channel.
+
+In the BabbleSim documentation you can find more information on how to
+`get <https://babblesim.github.io/fetching.html>`_. and
+`build <https://babblesim.github.io/building.html>`_ the simulator.
+In the :ref:`nrf52_bsim<nrf52_bsim>` board documentation you can find more information about how
+to build Zephyr targeting that particular board, and a few examples.
+
+Types of tests
+**************
+
+Tests without radio activity: bsim tests with twister
+-----------------------------------------------------
+
+The bsim boards can be used without radio activity, and in that case, it is not necessary to connect
```

Comment:
```suggestion
The BabbleSim boards can be used without radio activity, and in that case, it is not necessary to connect
```

or 
```suggestion
The `*_bsim` boards can be used without radio activity, and in that case, it is not necessary to connect
```

Reply:
Uhm, what if I add a link in that "bsim boards" text to the [bsim_boards page](https://docs.zephyrproject.org/latest/boards/posix/doc/bsim_boards_design.html)?

Reply:
That could also work :)